### ISSUE TITLE
Skip link for docs (accessibility improvement)

### ISSUE THREAD

**Issue by M4trox** [11ty/eleventy#815]
**Is your feature request related to a problem? Please describe.**
Let's suppose you use a keyboard to navigate through Eleventy docs main page. In order to focus some element within main content (such as link) you have to tab through whole navigation on the left. It takes a while, because there are 57 links in the sidebar!

**Describe the solution you'd like**
Skip link. If you use tab key on the site, you'll be asked to:

- skip to main content
- skip to sidebar navigation

Skip link will show up only if tab key used.

**Describe alternatives you've considered**
I have not considered any alternatives ;)

**Additional context**
See how Google does it:
![skip-link-in-google](https://user-images.githubusercontent.com/3310268/70391136-2e5e9b00-19d2-11ea-99e9-6f358d85394d.

@zachleat if you like the idea, I'll be happy to contribute :)

**Comment by zachleat** [comment]
Yes, this is very much needed! Moving this to 11ty-website repo.
